platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/BrowseCommand.java

```diff
@@ -63,7 +64,9 @@ public Integer call() throws IOException, InterruptedException {
             return FATAL_ERROR.getExitCode();
         }
 
-        Browser.join();
-        return BROWSER_WINDOW_CLOSED.getExitCode();
+        // Sleep forever to keep the process alive.
+        while (true) {
+            MINUTES.sleep(1);
```

Comment:
You removed the join in browser, that is probably what the problem is

Reply:
I added the join in `Browser` about a week ago. Not too worried about breaking old workflows by removing it. :)

We currently do not have a well defined node teardown workflow. I think we should eventually have it, but that's going to require some big changes.

In the meantime, this ugly little while loop seems to work well enough when running local demos. Everything still closes down very quickly if you press `ctrl-c` or if you close the main browser window.

Reply:
but if you run `java -jar swirlds.jar` will it end as soon as the main method ends? I think it will....